packages/babel-plugin-transform-runtime/test/fixtures/absoluteRuntime/relative/output.js

```diff
@@ -1,4 +1,4 @@
-var _classCallCheck = require("<CWD>/packages/babel-plugin-transform-runtime/node_modules/@babel/runtime/helpers/classCallCheck");
+var _classCallCheck = require("<CWD>/packages/babel-plugin-transform-runtime/test/node_modules/@babel/runtime/helpers/classCallCheck");
```

Comment:
🤔The extra `test` here looks strange.

Reply:
It's because I created a "fake" node_modules folder inside of `/test/`, to avoid relying on a "real" `@babel/runtime` which isn't part of the test suite and whose version will change.
For this reason, `transform-runtime` is resolving this new package.

Reply:
Ahh, I see. Thank you for the explainer.